### PR TITLE
Adds Centred SS13 Floorsigns

### PR DESCRIPTION
**Pull request by comradef191**
## About the PR 

This simply adds two centred versions of the 'Space Station 13' Floorsign seen on multiple maps. 

## Why's this needed? 
This just adds some extra options for those mapping

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,6 +32,29 @@ Build System.dm
...
+maps/WIP/Donut1/Donut1Station.dmm
+maps/WIP/backup/Artemis_Submiss1-Old_Arrival_shut.dmm
+maps/WIP/stat1.dmm
+maps/WIP/singustation13.dmm
+maps/WIP/NSS-Raven.dmm
+maps/WIP/minimap.dmm
+maps/WIP/Shuttles.dmm
+maps/WIP/5-6Station.dmm
+maps/WIP/4station.dmm
+maps/WIP/FartmapMocup.dmm
```
Comment (pali6) [inline .gitignore:57]:
```suggestion
```
I don't think you meant to commit this?

File: goonstation.dme [hunk 2]
```diff
@@ -50,6 +50,32 @@ var/datum/preMapLoad/preMapLoad = new
...
+#include "_stdlib\code\_controls.dm"
+#include "_stdlib\code\_dmm_suite.dm"
+#include "_stdlib\code\_fluid_defines.dm"
+#include "_stdlib\code\_layer.dm"
+#include "_stdlib\code\_macros.dm"
+#include "_stdlib\code\_plane.dm"
+#include "_stdlib\code\_vox.dm"
+#include "_stdlib\code\_z_material_properties.dm"
+#include "_stdlib\code\spaceman_dmm.dm"
+#include "code\__secret.dm"
```
Comment (pali6) [inline goonstation.dme:78]:
```suggestion
```
Nor this.

File: .gitignore (hunk 1 above)
Comment (comradef191) [reply .gitignore:57]: aaaaaa god no

File: goonstation.dme (hunk 2 above)
Comment (comradef191) [reply goonstation.dme:78]: god damn it

**Pull request merged**